src/FilamentPasswordlessPlugin.php

```diff
@@ -0,0 +1,56 @@
+<?php
+ 
+namespace BradyRenting\FilamentPasswordless;
+ 
+use BradyRenting\FilamentPasswordless\Http\Livewire\Auth\Login;
+use Filament\Contracts\Plugin;
+use Filament\Panel;
+use Illuminate\Foundation\Auth\User as Authenticatable;
```

Comment:
```suggestion
use Illuminate\Contracts\Auth\Authenticatable
```

This should use the Laravel `Authenticatable` interface (contract) instead of the concrete user model as it is perfectly fine to have a model that does not extend this class, but instead implement the interface.